### ISSUE TITLE
How to set array directly to iniFile and save?

### ISSUE THREAD

**Issue by LufoX11** [retrinko/ini#1]
Hi. I have an array with different sections and I need to be able to directly set an IniFile instance and save it. For now, my only solution was to iterate and set each section. Am I missing something? A function like: $iniFile->load($dataAsArray) would be perfect (I know load() is waiting for a file path. maybe adding a condition for letting also get arrays as well...). Thanks.

**Comment by retrinko** [comment]
Hi @shamank,

Check the last release I have published a couple of hours ago (v0.3). 

The new class _Retrinko\Ini\IniFile\Factory_ will allow you to get an instance of _IniFile_ from a data array or an array of _IniSection_ instances.

I hope that helps you! 

A little sample: 

``` php
    // Data array
    $data = ['section 1' => ['key1' => 'value 1.1',
                             'key2' => 'value 1.2'],
             'section 2' => ['key3' => 'value 2.3'],
             'section 3 : section 1' => ['key1' => 'value 3.1'],
             'section 4' => ['intVal' => 1,
                             'floatVal' => 1.2,
                             'boolVal' => false,
                             'nullVal' => null,
                             'arrayVal' => ['a'=>'A', 'b'=>'B', 'c'=>'C']]
    ];
    // Get an IniFile instance from the previous array
    $iniFile = IniFile\Factory::fromArray($data);

    // Save to file
    $iniFile->save(__DIR__ . '/sample.ini');
```

You can also check the sample file: _/samples/load-array.php_

**Comment by LufoX11** [comment]
It worked perfect, thanks!
